index.html

```diff
@@ -1,81 +1,81 @@
 <!DOCTYPE html>
-<html lang = en>
-<head>
-	<meta charset="utf-8">
-	<title>Rocket Science</title>
-</head>
-<body>
-	<div id = "branding">
-		<img src = "" alt = "logo"/>
-		<h1>Rocket Science</h1>
-		<h2>Software Craftsmanship</h2>
-	</div>
+<html lang=en>
+  <head>
+    <meta charset="utf-8">
+    <title>Rocket Science</title>
+  </head>
+  <body>
+    <div id="branding">
+      <img src="" alt="logo"/>
+      <h1>Rocket Science</h1>
+      <h2>Software Craftsmanship</h2>
+    </div>
 
-	<div id = "introText">
-		<p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut. Ex eligendi insolens sapientem per, duo id tale sumo propriae, ut falli inermis cum.</p>
-	</div>
+    <div id="introText">
+      <p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut. Ex eligendi insolens sapientem per, duo id tale sumo propriae, ut falli inermis cum.</p>
+    </div>
 
-	<hr>
+    <hr>
```

Comment:
lets scrap this guy - `<hr>` is an oldskool presentational element that we shouldn't need to use any more. We'll accomplish this by adding a `border-bottom` to the `<div>` above (or something similar)!

Reply:
reason for this is to keep ALL of our styling in css and use HTML ONLY for document structure - a horizontal rule or `<hr>` is a stylistic touch, and should be expressed in CSS to keep things tidy.